Dev-Corner/slides/tool_integration.html

```diff
@@ -45,65 +45,1274 @@
+                        <script type="text/template">
+                          ### `tests`
+                          #### Comparison Methods within the output
+
+                          ```xml
+                          <test>
+                              <output>
+                                  <assert_contents>
+                                      <has_text />
+                                      <not_has_text />
+                                      <has_text_matching />
+                                      <has_text />
+                                      <...>
+                          ```
+                          <small>[https://docs.galaxyproject.org/en/latest/dev/schema.html#tool|tests|test|output|assert_contents](https://docs.galaxyproject.org/en/latest/dev/schema.html#tool|tests|test|output|assert_contents)</small>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/planemo-logo.png" alt="Planemo" class="no-border" height=25/>
+                            #### Test Galaxy Tools
+
+                            <img src="../images/planemo_tool_building_test.png" alt="Testing tools" class="no-border"/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            <img src="../images/exercise.png" alt="exercise"/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            #### The Goal
+                            At functional test to the Seqtk wrapper for Seqtk
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            #### Steps
+                            1. Fill a `<test>` tag using the file you generate previously with `seqtk`
+                            2. Use `planemo test --install_galaxy ...` until all is green
+
+                        </script>
+                    </section>
+                </section>
+
+                <!-- DEPENDENCIES -->
+                <section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ## Dependencies
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Dependencies
+                            #### What about tool installation?
```

Comment:
Only "What about tool installation?" as title? What do you think?
Same questions for the next slides